README.md

```diff
@@ -1,4 +1,4 @@
-# Lux
+# Lux [![Build Status](https://travis-ci.org/postlight/lux.svg?branch=master)](https://travis-ci.org/postlight/lux)
```

Comment:
Would also be cool to add the badges via [Shields.io](http://shields.io). You could also get the nifty NPM badges if you wanted.